internal/controller/storagemgr/config/toml_test.go

```diff
@@ -0,0 +1,42 @@
+/*******************************************************************************
+ * Copyright 2021 Samsung Electronics All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ *******************************************************************************/
+
+package config
+
+import (
+	"github.com/lf-edge/edge-home-orchestration-go/internal/common/logmgr"
+	"testing"
+)
+
+var (
+	log = logmgr.GetInstance()
+)
+
+func TestToml(t *testing.T) {
+	SetWritable("DEBUG")
+	SetService(49986, nil)
+	SetRegistry("127.0.0.1", 8500)
```

Comment:
@t25kim Just for a check: whether or not it triggers https://github.com/lf-edge/edge-home-orchestration-go/issues/273. Thank you! 😄

Reply:
@MoonkiHong Thanks for review! Don't worry. SonarCloud does not check with test files.

Reply:
@t25kim Confirmed! Thank you!! 😁